### PR TITLE
Fixing event-scraper so that XHTML is saved with correct UTF-8 charac…

### LINKED ISSUE #7: [bug] event-scraper does not store event XHTML files with correct UTF-8 character encoding
example:  http://webmontag.de/location/koeln/2005-11-07 => `"Jörg Heidjann"`

### PR DESCRIPTION
**Pull request by amed**
…ter encoding, so that closes #7

### CODE REVIEW

File: lib/events-scraper/event.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def remote_document
       uri = "#{WEBSITE}#{@link}?do=export_xhtmlbody"
       print "\n #{@index+1} ".yellow
       print "#{WEBSITE}#{@link}.html ".blue
-      Nokogiri::HTML(open(uri))
+      Nokogiri::HTML(open(uri), nil,"UTF-8")
```
Comment (jhilden) [inline lib/events-scraper/event.rb:68]: the last argument wants some space to breathe.  Pacman!

(I'm wondering if `UTF-8` is not the default anyway?)

**Review by jhilden — COMMENTED** [review]
very good.  feel free to merge it

File: lib/events-scraper/event.rb (hunk 1 above)
Comment (amed) [reply lib/events-scraper/event.rb:68]: Yes it is the default value.

Comment (jhilden) [reply lib/events-scraper/event.rb:68]: in that case, would it be enough to just set UTF-8 when writing the output file?

Comment (amed) [reply lib/events-scraper/event.rb:68]: Oh god, the test fails. I thought also it should be default UTF-8.

**Comment by jhilden** [comment]
Deploy preview ready!

Built with commit 0ae8a4cdb0247a4f5780767f78d138a7fd35ce75

https://deploy-preview-10--webmontag.netlify.com

**Pull request merged**